### DISCUSSION TITLE
How to install the latest version of GrandOrgue on Ubuntu

### DISCUSSION THREAD

**Discussion by vpoguru** [GrandOrgue/grandorgue#1478] [Help]
I am trying to install the latest version of GrandOrgue from Github on Ubuntu. When downloading and double clicking the deb image version 3.9.5-debian is still installed.

When I install the package in the terminal with 'sudo dpkg -i grandorgue_3.10.1-1_amd64.deb', I get the error message that two dependencies are missing: libwxbase3.0-0v5 (>= 3.0.4+dfsg) and libwxgtk3.0-gtk3-0v5 (>= 3.0.4+dfsg).

Trying to install libwxbase3.0-0v5 with 'sudo apt-get install -y libwxbase3.0-0v5', as described [here](https://zoomadmin.com/HowToInstall/UbuntuPackage/libwxbase3.0-0v5) gives the error message that no installation candidate exists for package "libwxbase3.0-0v5".

**Comment by oleg68** [comment]
What ubuntu version do you have?

**Comment by vpoguru** [comment]
I had the problem with Ubuntu 22.10. Yesterday my computer wanted to upgrade to 23.04. The problem still exists.

**Reply by oleg68** [reply]
In ubuntu 22.10 the next way works:

```
apt-get update
apt-get install /home/oleg/Downloads/grandorgue_3.10.1-1_amd64.deb
```

Unfortunally, it doesn't work in ubuntu 23.04, because there is no wxgtk3.0 in this distro. So you have two options
1. At first install grandorgue in 22.10 and at second upgrade to 23.04
2. Use  `.Appimage`.

**Comment by vpoguru** [comment]
Many thanks! The Apimmage works. It is always a bit cumbersome to add an app image to the dock.

Any chance that GO is updated to work with wxgtk3.2 in the near future?

**Reply by oleg68** [reply]
It isn't so difficult to compile GO for working with wxgtk3.2. The problem is to create a single package that can work with either 3.0 or 3.2. But old linux distros, including Ubuntu 20, do not provide 3.2, so compatibility with 3.0 is still necessary.

**Reply by oleg68** [reply]
I created an issue #1480
